lib/coinbase/wallet.rb

```diff
@@ -182,11 +195,17 @@ def faucet
       end
     end
 
+    # Returns whether the Wallet has a seed with which to derive keys and sign transactions.
+    # @return [Boolean] Whether the Wallet has a seed with which to derive keys and sign transactions.
+    def seed?
+      !@master.nil?
+    end
+
     # Returns a String representation of the Wallet.
     # @return [String] a String representation of the Wallet
     def to_s
       "Coinbase::Wallet{wallet_id: '#{id}', network_id: '#{network_id}', " \
-        "default_address: '#{default_address.id}'}"
+        "default_address: '#{@model.default_address.address_id}'}"
```

Comment:
Since `default_address` can be nil, do we want `@model.default_address&.address_id`?

Reply:
Done